cmd/osv-scanner/main.go

```diff
@@ -89,6 +89,11 @@ func run(args []string, stdout, stderr io.Writer) int {
 				Usage:   "check subdirectories",
 				Value:   false,
 			},
+			&cli.BoolFlag{
+				Name:  "ignore-gitignore",
```

Comment:
Suggestions for an alternative name for this flag would be appreciated

Reply:
`ripgrep` uses the flag `--no-ignore`, so that could be an alternative name.

Reply:
Changed to `--no-ignore` now